DuckDuckGo/SettingsHostingController.swift

```diff
@@ -48,9 +48,10 @@ class SettingsHostingController: UIHostingController<AnyView> {
         }
 
         let settingsView: AnyView
-        switch PixelExperiment.cohort {
-        case .control, .noVariant, .none: settingsView = AnyView(SettingsView(viewModel: viewModel))
-        case .newSettings: settingsView = AnyView(SettingsRootView(viewModel: viewModel))
+        if PixelExperiment.cohort == .newSettings {
```

Comment:
if you're gonna make this kind of change, a ternary is more concise imo

    let settingsView = PixelExperiment.cohort == .newSettings ? 
         AnyView(SettingsRootView(viewModel: viewModel)) : 
         AnyView(SettingsView(viewModel: viewModel))